SA3/qa_web/static/js/base.js

```diff
@@ -1,5 +1,24 @@
 /*JAVASCRIPT for QA_Web*/
 
+$(document).ready(function(){
```

Comment:
Is there a specific reason this is not in the same $(document).ready call that was already present in base.js?

Reply:
You're right, I didn't see the other one!